examples/jsm/renderers/webgpu/WebGPUAttributes.js

```diff
@@ -117,7 +119,7 @@ class WebGPUAttributes {
 	_createBuffer( attribute, usage ) {
 
 		const array = attribute.array;
-		const size = array.byteLength + ( ( 4 - ( array.byteLength % 4 ) ) % 4 ); // ensure 4 byte alignment, see #20441
+		const size = array.byteLength + MathUtils.euclideanModulo( - array.byteLength, 4 ); // ensure 4 byte alignment
```

Comment:
I find it a bit strange that `euclideanModulo()` is always called with a negative `n` value. Any chances to rewrite this?

Reply:
This is exactly what allows to simplify -- `( - m ) % n` almost always equals `n - ( m % n )`, except when `m % n` is 0 -- then it is also 0. So alignment calculation simplifies from something like `r = m % n; return r === 0 ? m : m + n - r` to just `m + ( - m ) % n`. I suggest we turn it in a function like `MathUtils.align` if it's too hard to read.

Reply:
The simplest expression is maybe `Math.ceil( m / n ) * n`.

Reply:
@Mugen87 Should I go with creating a function in MathUtils returning `Math.ceil( m / n ) * n`? How should we call it?

Reply:
@Mugen87 What do you think?